PyNite/Member3D.py

```diff
@@ -330,85 +331,84 @@ def T(self):
         z2 = self.jNode.Z
         L = self.L()
         
-        # The following commented code is no longer used to formulate the transformation matrix.
-        # It was found to be incorrect, even though it follows 'A First Course in the Finite Element Method, 4th Ed.'.
-        # It's being kept for reference for now. The 'z' direction cosines in it are not a unit vector.
-        # It also keeps the local y-axis perpendicular to the global Z-axis, which isn't very intuitive
-        # for the user. It can lead to unexpected member local axis 'flipping'. A better formulation has
-        # been implemented below.
-
-        # # Calculate direction cosines for the transformation matrix
-        # if isclose(x2-x1, 0.0) and isclose(y2-y1, 0.0) and (z2-z1 > 0.0):
-        #     dirCos = matrix([[0, 0, 1],
-        #                      [0, 1, 0],
-        #                      [-1, 0, 0]])
-        # elif isclose(x2-x1, 0.0) and isclose(y2-y1, 0.0) and (z2-z1 < 0.0):
-        #     dirCos = matrix([[0, 0, -1],
-        #                      [0, 1, 0],
-        #                      [1, 0, 0]])
-        # else:
-        #     l = (x2-x1)/L
-        #     m = (y2-y1)/L
-        #     n = (z2-z1)/L
-        #     D = (l**2+m**2)**0.5
-        #     dirCos = matrix([[l, m, n],
-        #                      [-m/D, l/D, 0],
-        #                      [-l*n/D, -m*n/D, D]])
-
         # Calculate the direction cosines for the local x-axis
         x = [(x2-x1)/L, (y2-y1)/L, (z2-z1)/L]
```

Comment:
I think if user use auxiliary node for every member, we reduce probability of errors in the model. Using auxiliary node, user knows exactly the orientation of local refering system. Because of thi I've deleted te previously formulation of T matrix

Reply:
I've set up the default behavior to orient members just like most commercial programs do. PyNite behaves just like STAAD, RISA, RAM and ROBOT, so most users should be used to this behavior. For users who want more control, we now have the auxiliary node option thanks to your contribution.